opentelemetry-resource-detectors/README.md

```diff
@@ -0,0 +1,18 @@
+![OpenTelemetry — An observability framework for cloud-native software.][splash]
+
+[splash]: https://raw.githubusercontent.com/open-telemetry/opentelemetry-rust/main/assets/logo-text.png
+
+# OpenTelemetry Contrib Resource Detectors
+
+Community supported Resource detectors implementations for applications instrumented with [`OpenTelemetry`].
+
+## Overview
+
+[`The contrib resource detector`] crate provides a collection of tools to detect resources from the runtime. This crate provides additional detectors for OS and process-related resources. Despite not being included in the SDK due to their lack of stability, these detectors can facilitate gathering additional telemetry information.
```

Comment:
nit - `opentelemetry-resource-detectors` crate

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/49/commits/98052034707af7285864ea267c899b9b8b632e08
Thank you